cli/azd/pkg/project/scaffold_gen.go

```diff
@@ -339,6 +339,13 @@ func mapHostUses(
 			backendMapping[use] = res.Name // record the backend -> frontend mapping
 		case ResourceTypeOpenAiModel:
 			svcSpec.AIModels = append(svcSpec.AIModels, scaffold.AIModelReference{Name: use})
+		case ResourceTypeMessagingServiceBus:
+			props := useRes.Props.(ServiceBusProps)
+			svcSpec.AzureServiceBus = &scaffold.AzureDepServiceBus{
+				Queues:   props.Queues,
+				AuthType: props.AuthType,
+				IsJms:    props.IsJms,
+			}
```

Comment:
Add event-hubs & storage-accounts here.
But you can ignore it in current PR because I refactored this code in my PR: https://github.com/azure-javaee/azure-dev/pull/30